docs/content/concepts/scheduler.md

```diff
@@ -76,6 +64,25 @@ the same number of tokens.
 
 :::
 
+![Token Bucket](./assets/img/token-bucket-light.svg#gh-light-mode-only)
+![Token Bucket](./assets/img/token-bucket-dark.svg#gh-dark-mode-only)
+
+The diagram at hand illustrates the use of the token bucket algorithm for
+workload prioritization by the scheduler. It demonstrates how the Aperture
+Controller periodically adjusts the token bucket's load multiplier and refill
+rate by broadcasting signals to the Aperture Agent.
+
+In the scope of the Aperture Agent, the scheduler efficiently handles incoming
+requests, categorizing them according to their urgency. This classification
+scale ranges from 0 to 255, where 0 indicates the lowest and 255 the highest
```

Comment:
Priority format and meaning has changed in latest

Reply:
@sudhanshu456 You can check https://docs.fluxninja.com/development/reference/configuration/spec#scheduler-workload-parameters